### PR TITLE
Adds CI tests for Ubuntu image

### PR DESCRIPTION
**Pull request by yguo0905**
Added the following tests

ci-kubernetes-e2e-gce-ubuntu
ci-kubernetes-e2e-gce-ubuntu-serial
ci-kubernetes-e2e-gce-ubuntu-slow

### CODE REVIEW

**Comment by yguo0905** [comment]
@krzyzacy @dchen1107

**Comment by dchen1107** [comment]
#2592 was merged.

@k8s-bot bazel test this

**Comment by fejta** [comment]
Very cool and very exciting. Thank you so much for adding this to prow instead of jenkins!!

File: jobs/config.json [hunk 1]
```diff
@@ -3310,5 +3310,30 @@
...
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-e2e-gce-ubuntu-serial.env"
+  ]
+},
+
+"ci-kubernetes-e2e-gce-ubuntu-slow": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-kubernetes-e2e-gce-ubuntu-slow.env"
```
Comment (krzyzacy) [inline jobs/config.json]: prow needs `--mode=local`

**Pull request merged**